  INFO root [2020-10-25 19:20:50,677] - 
  INFO root [2020-10-25 19:20:50,680] - -----------------------------------------------------------------------------------------


  INFO root [2020-10-25 19:20:50,681] - Resmgr::initialize: KBE_ROOT=D:/kbengine_24pic/
  INFO root [2020-10-25 19:20:50,682] - Resmgr::initialize: KBE_RES_PATH=D:/kbengine_24pic/kbe/res/;D:/kbengine_24pic/kbengine_stone_assets/;D:/kbengine_24pic/kbengine_stone_assets/scripts/;D:/kbengine_24pic/kbengine_stone_assets/res/
  INFO root [2020-10-25 19:20:50,682] - Resmgr::initialize: KBE_BIN_PATH=D:/kbengine_24pic/kbe/bin/server/
  INFO root [2020-10-25 19:20:50,690] - NetworkInterface::initialize(EXTERNAL): bound to all interfaces with default route interface on 192.168.2.246 ( 192.168.2.246:20013 )
  INFO root [2020-10-25 19:20:50,691] - NetworkInterface::initialize(EXTERNAL): address 192.168.2.246:20013, SOMAXCONN=128.
  INFO root [2020-10-25 19:20:50,695] - NetworkInterface::initialize(INTERNAL): bound to all interfaces with default route interface on 192.168.2.246 ( 192.168.2.246:51617 )
  INFO root [2020-10-25 19:20:50,696] - NetworkInterface::initialize(INTERNAL): address 192.168.2.246:51617, SOMAXCONN=128.
  INFO root [2020-10-25 19:20:50,696] - server-configs:
  INFO root [2020-10-25 19:20:50,696] - 	internalAddr : 192.168.2.246:51617
  INFO root [2020-10-25 19:20:50,697] - 	externalAddr : 192.168.2.246:20013
  INFO root [2020-10-25 19:20:50,697] - 	componentID : 920002
  INFO root [2020-10-25 19:20:51,094] - Components::findLogger: found logger, addr:192.168.2.246:51613
  INFO root [2020-10-25 19:20:51,094] - Components::addComponent: reset orderLog, uid=11640!
  INFO root [2020-10-25 19:20:51,094] - Components::addComponent[logger], uid=11640, componentID=210002, globalorderid=1, grouporderid=1, totalcount=1
  INFO root [2020-10-25 19:20:51,099] - ---- loginapp Version: 1.2.7. ScriptVersion: 0.1.0. Protocol: A87D1428E6DE7B1742DD55BD6E5BAE8C. Config: Debug X86. Built: 19:01:57 Aug  4 2020. AppID: 920002. UID: 11640. PID: 7724 ----
  INFO root [2020-10-25 19:20:51,099] - Server : Intel(R) Core(TM) i5-10210U CPU @ 1.60GHz with 2097151kB RAM
  INFO root [2020-10-25 19:20:51,100] - ThreadPool::createThreadPool: creating  threadpool...
  INFO root [2020-10-25 19:20:51,100] - ThreadPool::createThreadPool: successfully(2), newThreadCount=1, normalMaxThreadCount=2, maxThreadCount=8
 DEBUG root [2020-10-25 19:20:51,231] - Script::install_py_dlls(): PyInit_pyexpat
 DEBUG root [2020-10-25 19:20:51,232] - Script::install_py_dlls(): PyInit__socket
 DEBUG root [2020-10-25 19:20:51,232] - Script::install_py_dlls(): PyInit__ssl
 DEBUG root [2020-10-25 19:20:51,237] - Script::install_py_dlls(): PyInit__hashlib
 DEBUG root [2020-10-25 19:20:51,237] - Script::install_py_dlls(): PyInit_select
 DEBUG root [2020-10-25 19:20:51,239] - Script::install_py_dlls(): PyInit__ctypes
 DEBUG root [2020-10-25 19:20:51,240] - Script::install_py_dlls(): PyInit__elementtree
 DEBUG root [2020-10-25 19:20:51,273] - Script::install_py_dlls(): PyInit_unicodedata
  INFO root [2020-10-25 19:20:51,304] - Script::install(): is successfully, Python=(3.4.2 (default, Aug  4 2020, 19:01:54) [MSC v.1925 32 bit (Intel)])!
  INFO root [2020-10-25 19:20:51,304] - Script::install(): KBExtra is successfully!
S_INFO root [2020-10-25 19:20:51,313] - onLoginAppReady: bootstrapGroupIndex=0, bootstrapGlobalIndex=0
  INFO root [2020-10-25 19:20:51,313] - TelnetServer server is running on port 31000
  INFO root [2020-10-25 19:20:51,543] - ---- loginapp is running ----
 DEBUG root [2020-10-25 19:20:51,543] - Components::process(): Request for the process of identity...
  WARN root [2020-10-25 19:20:57,710] - Forwarding logs to logger[192.168.2.246:51613/0/0/0]...
 ERROR root [2020-10-25 19:27:50,778] - Components::removeComponentByChannel: logger : 210002, Abnormal exit! TCPPacketReceiver::processRecv(): errno=10054

  INFO root [2020-10-25 19:27:50,781] - The following logs sent to logger failed:
  WARN root [2020-10-25 19:27:50,782] - ==> [2020-10-25 19:27:50 777] TCPPacketReceiver::processPendingEvents(192.168.2.246:51613): Throwing REASON_GENERAL_NETWORK - WSAECONNRESET
 ERROR root [2020-10-25 19:27:50,785] - ==> [2020-10-25 19:27:50 779] Components::removeComponentByChannel: logger : 210002, Abnormal exit! TCPPacketReceiver::processRecv(): errno=10054

  WARN root [2020-10-25 19:27:50,904] - TCPPacketReceiver::processPendingEvents(192.168.2.246:51614): Throwing REASON_GENERAL_NETWORK - WSAECONNRESET
 ERROR root [2020-10-25 19:27:50,907] - Components::removeComponentByChannel: dbmgr : 42002, Abnormal exit! TCPPacketReceiver::processRecv(): errno=10054

  INFO root [2020-10-25 19:27:50,938] - Shutdowner::onShutdownBegin: shutting down(period=0, tickPeriod=1)
S_INFO root [2020-10-25 19:27:50,944] - onLoginAppShutDown()
  INFO root [2020-10-25 19:27:50,946] - Shutdowner::onShutdown: shutting down
  INFO root [2020-10-25 19:27:50,946] - Shutdowner::onShutdownEnd: shutting down
 DEBUG root [2020-10-25 19:27:50,948] - Script::uninstall_py_dlls(): PyInit_pyexpat
 DEBUG root [2020-10-25 19:27:50,949] - Script::uninstall_py_dlls(): PyInit__socket
 DEBUG root [2020-10-25 19:27:50,950] - Script::uninstall_py_dlls(): PyInit__ssl
 DEBUG root [2020-10-25 19:27:50,951] - Script::uninstall_py_dlls(): PyInit__hashlib
 DEBUG root [2020-10-25 19:27:50,952] - Script::uninstall_py_dlls(): PyInit_select
 DEBUG root [2020-10-25 19:27:50,952] - Script::uninstall_py_dlls(): PyInit__ctypes
 DEBUG root [2020-10-25 19:27:50,953] - Script::uninstall_py_dlls(): PyInit__elementtree
 DEBUG root [2020-10-25 19:27:50,954] - Script::uninstall_py_dlls(): PyInit_unicodedata
  INFO root [2020-10-25 19:27:50,994] - Script::uninstall(): is successfully!
 DEBUG root [2020-10-25 19:27:50,994] - ThreadPool::destroy(): starting size 2.
  WARN root [2020-10-25 19:27:51,295] - ThreadPool::destroy(): waiting for thread(2)[0x22659d8,0x2265b78,], try=1
  INFO root [2020-10-25 19:27:54,680] - 
  INFO root [2020-10-25 19:27:54,681] - -----------------------------------------------------------------------------------------


  INFO root [2020-10-25 19:27:54,682] - Resmgr::initialize: KBE_ROOT=D:/kbengine_24pic/
  INFO root [2020-10-25 19:27:54,684] - Resmgr::initialize: KBE_RES_PATH=D:/kbengine_24pic/kbe/res/;D:/kbengine_24pic/kbengine_stone_assets/;D:/kbengine_24pic/kbengine_stone_assets/scripts/;D:/kbengine_24pic/kbengine_stone_assets/res/
  INFO root [2020-10-25 19:27:54,684] - Resmgr::initialize: KBE_BIN_PATH=D:/kbengine_24pic/kbe/bin/server/
  INFO root [2020-10-25 19:27:54,689] - NetworkInterface::initialize(EXTERNAL): bound to all interfaces with default route interface on 192.168.2.246 ( 192.168.2.246:20013 )
  INFO root [2020-10-25 19:27:54,690] - NetworkInterface::initialize(EXTERNAL): address 192.168.2.246:20013, SOMAXCONN=128.
  INFO root [2020-10-25 19:27:54,693] - NetworkInterface::initialize(INTERNAL): bound to all interfaces with default route interface on 192.168.2.246 ( 192.168.2.246:51748 )
  INFO root [2020-10-25 19:27:54,694] - NetworkInterface::initialize(INTERNAL): address 192.168.2.246:51748, SOMAXCONN=128.
  INFO root [2020-10-25 19:27:54,694] - server-configs:
  INFO root [2020-10-25 19:27:54,694] - 	internalAddr : 192.168.2.246:51748
  INFO root [2020-10-25 19:27:54,694] - 	externalAddr : 192.168.2.246:20013
  INFO root [2020-10-25 19:27:54,694] - 	componentID : 920002
  INFO root [2020-10-25 19:27:54,901] - Components::findLogger: found logger, addr:192.168.2.246:51741
  INFO root [2020-10-25 19:27:54,902] - Components::addComponent: reset orderLog, uid=13025!
  INFO root [2020-10-25 19:27:54,902] - Components::addComponent[logger], uid=13025, componentID=210002, globalorderid=1, grouporderid=1, totalcount=1
  INFO root [2020-10-25 19:27:54,907] - ---- loginapp Version: 1.2.7. ScriptVersion: 0.1.0. Protocol: A87D1428E6DE7B1742DD55BD6E5BAE8C. Config: Debug X86. Built: 19:01:57 Aug  4 2020. AppID: 920002. UID: 13025. PID: 11060 ----
  INFO root [2020-10-25 19:27:54,907] - Server : Intel(R) Core(TM) i5-10210U CPU @ 1.60GHz with 2097151kB RAM
  INFO root [2020-10-25 19:27:54,907] - ThreadPool::createThreadPool: creating  threadpool...
  INFO root [2020-10-25 19:27:54,908] - ThreadPool::createThreadPool: successfully(2), newThreadCount=1, normalMaxThreadCount=2, maxThreadCount=8
 DEBUG root [2020-10-25 19:27:55,037] - Script::install_py_dlls(): PyInit_pyexpat
 DEBUG root [2020-10-25 19:27:55,037] - Script::install_py_dlls(): PyInit__socket
 DEBUG root [2020-10-25 19:27:55,038] - Script::install_py_dlls(): PyInit__ssl
 DEBUG root [2020-10-25 19:27:55,040] - Script::install_py_dlls(): PyInit__hashlib
 DEBUG root [2020-10-25 19:27:55,041] - Script::install_py_dlls(): PyInit_select
 DEBUG root [2020-10-25 19:27:55,041] - Script::install_py_dlls(): PyInit__ctypes
 DEBUG root [2020-10-25 19:27:55,041] - Script::install_py_dlls(): PyInit__elementtree
 DEBUG root [2020-10-25 19:27:55,070] - Script::install_py_dlls(): PyInit_unicodedata
  INFO root [2020-10-25 19:27:55,101] - Script::install(): is successfully, Python=(3.4.2 (default, Aug  4 2020, 19:01:54) [MSC v.1925 32 bit (Intel)])!
  INFO root [2020-10-25 19:27:55,101] - Script::install(): KBExtra is successfully!
S_INFO root [2020-10-25 19:27:55,104] - onLoginAppReady: bootstrapGroupIndex=0, bootstrapGlobalIndex=0
  INFO root [2020-10-25 19:27:55,105] - TelnetServer server is running on port 31000
  INFO root [2020-10-25 19:27:55,426] - ---- loginapp is running ----
 DEBUG root [2020-10-25 19:27:55,427] - Components::process(): Request for the process of identity...
  WARN root [2020-10-25 19:28:01,595] - Forwarding logs to logger[192.168.2.246:51741/0/0/0]...
 ERROR root [2020-10-25 23:03:02,839] - Components::removeComponentByChannel: logger : 210002, Abnormal exit! TCPPacketReceiver::processRecv(): errno=10054

  INFO root [2020-10-25 23:03:02,861] - The following logs sent to logger failed:
  WARN root [2020-10-25 23:03:02,864] - ==> [2020-10-25 23:03:02 833] TCPPacketReceiver::processPendingEvents(192.168.2.246:51741): Throwing REASON_GENERAL_NETWORK - WSAECONNRESET
 ERROR root [2020-10-25 23:03:02,865] - ==> [2020-10-25 23:03:02 857] Components::removeComponentByChannel: logger : 210002, Abnormal exit! TCPPacketReceiver::processRecv(): errno=10054

  WARN root [2020-10-25 23:03:02,917] - TCPPacketReceiver::processPendingEvents(192.168.2.246:51743): Throwing REASON_GENERAL_NETWORK - WSAECONNRESET
 ERROR root [2020-10-25 23:03:02,920] - Components::removeComponentByChannel: dbmgr : 42002, Abnormal exit! TCPPacketReceiver::processRecv(): errno=10054

  INFO root [2020-10-25 23:03:02,927] - Shutdowner::onShutdownBegin: shutting down(period=0, tickPeriod=1)
S_INFO root [2020-10-25 23:03:02,933] - onLoginAppShutDown()
  INFO root [2020-10-25 23:03:02,933] - Shutdowner::onShutdown: shutting down
  INFO root [2020-10-25 23:03:02,934] - Shutdowner::onShutdownEnd: shutting down
 DEBUG root [2020-10-25 23:03:02,952] - Script::uninstall_py_dlls(): PyInit_pyexpat
 DEBUG root [2020-10-25 23:03:02,954] - Script::uninstall_py_dlls(): PyInit__socket
 DEBUG root [2020-10-25 23:03:02,955] - Script::uninstall_py_dlls(): PyInit__ssl
 DEBUG root [2020-10-25 23:03:02,956] - Script::uninstall_py_dlls(): PyInit__hashlib
 DEBUG root [2020-10-25 23:03:02,956] - Script::uninstall_py_dlls(): PyInit_select
 DEBUG root [2020-10-25 23:03:02,957] - Script::uninstall_py_dlls(): PyInit__ctypes
 DEBUG root [2020-10-25 23:03:02,957] - Script::uninstall_py_dlls(): PyInit__elementtree
 DEBUG root [2020-10-25 23:03:02,957] - Script::uninstall_py_dlls(): PyInit_unicodedata
  INFO root [2020-10-25 23:03:02,987] - Script::uninstall(): is successfully!
 DEBUG root [2020-10-25 23:03:02,989] - ThreadPool::destroy(): starting size 2.
  WARN root [2020-10-25 23:03:03,290] - ThreadPool::destroy(): waiting for thread(2)[0x2271dd8,0x2271bd0,], try=1
